### PR TITLE
Fix for #1042

### PR DESCRIPTION
**Pull request by cjdreiss**
Wrapping the call to `_currentPosition` in a setTimeout so the dropdown has fully rendered before we check the size.

### CODE REVIEW

**Comment by cjdreiss** [comment]
Looks like a few tests failed in Travis.  I can't easily install yarn at work, I will have to try and home and see if it was my change that caused the failures.

File: src/ng-select/ng-dropdown-panel.component.ts [hunk 1]
```diff
@@ -182,24 +182,26 @@ export class NgDropdownPanelComponent implements OnInit, OnChanges, OnDestroy, A
...
-            this._renderer.removeClass(this._dropdown, BOTTOM_CSS_CLASS);
-            this._renderer.addClass(this._select, TOP_CSS_CLASS);
-            this._renderer.removeClass(this._select, BOTTOM_CSS_CLASS)
-        } else {
-            this._renderer.addClass(this._dropdown, BOTTOM_CSS_CLASS);
-            this._renderer.removeClass(this._dropdown, TOP_CSS_CLASS);
-            this._renderer.addClass(this._select, BOTTOM_CSS_CLASS);
-            this._renderer.removeClass(this._select, TOP_CSS_CLASS);
-        }
+        setTimeout(() => {
```
Comment (anjmao) [inline src/ng-select/ng-dropdown-panel.component.ts]: You can try using _windowService.setTimeout which was created to mock setTimeout easer and tests should pass again.

Comment (cjdreiss) [reply src/ng-select/ng-dropdown-panel.component.ts]: Thanks, that fixed it.

**Comment by anjmao** [comment]
fixes #1042

**Pull request merged**